ShinyStoneHotswap.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace PvPAdventure
+{
+    public class ShinyStoneHotswap : ModBuff
+    {
+        public override void SetStaticDefaults()
+        {
+            Main.debuff[Type] = true; // duh duh
+            Main.buffNoSave[Type] = true; // doesn't matter because of anti-hotswap rule applying upon rejoining
+            Main.buffNoTimeDisplay[Type] = false; // Show timer
+        }
+    }
+
+    public class ShinyStoneDisabler : ModPlayer
+    {
+        private bool hadShinyStoneLastFrame;
```

Comment:
Private fields should be prefixed with a leading underscore `_hadShinyStoneLastFrame`.